src/compression/compression_simd.rs

```diff
@@ -1,44 +1,46 @@
-
 use super::NUM_DOCS_PER_BLOCK;
 
-use libc::size_t;
-
 const COMPRESSED_BLOCK_MAX_SIZE: usize = NUM_DOCS_PER_BLOCK * 4 + 1; 
 
-extern {
-    fn compress_sorted_cpp(
-        data: *const u32,
-        output: *mut u8,
-        offset: u32) -> size_t;
-
-    fn uncompress_sorted_cpp(
-        compressed_data: *const u8,
-        output: *mut u32,
-        offset: u32) -> size_t;
-        
-    fn compress_unsorted_cpp(
-        data: *const u32,
-        output: *mut u8) -> size_t;
-
-    fn uncompress_unsorted_cpp(
-        compressed_data: *const u8,
-        output: *mut u32) -> size_t;
+mod simdcomp {
```

Comment:
There isn't really one, but since I removed the `_cpp` suffix on the wrapper functions putting the C functions in a module was the smallest change so they don't conflict with the other functions in there.